sys/suit/v3/handlers_container.c

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) 2019 Koen Zandberg
+ *               2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup     sys_suit_v3
+ * @{
+ *
+ * @file
```

Comment:
Same as for the above handlers, a little more context please :)

Reply:
Done